src/nora/components/NoraButton/index.js

```diff
@@ -0,0 +1 @@
+export { NoraButton } from './NoraButton'
```

Comment:
Can we write a hook or something that checks that these components are properly exported, or add this to a README somewhere?